src/Prometheus.Client.MetricServer.Kestrel/Prometheus.Client.MetricServer.Kestrel.csproj

```diff
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>MetricServer for the Prometheus.Client</Description>
+    <Copyright>2019 © Sergey Kuznetsov</Copyright>
+    <AssemblyTitle>Prometheus.Client.MetricServer</AssemblyTitle>
+    <VersionPrefix>3.0.2</VersionPrefix>
+    <Authors>Sergey Kuznetsov</Authors>
+    <TargetFrameworks>netstandard1.3</TargetFrameworks>
+    <AssemblyName>Prometheus.Client.MetricServer</AssemblyName>
+    <PackageId>Prometheus.Client.MetricServer</PackageId>
+    <PackageTags>prometheus;metrics</PackageTags>
+    <PackageIconUrl>https://image.ibb.co/k4Sc0k/prometheus.png</PackageIconUrl>
+    <PackageProjectUrl>https://github.com/PrometheusClientNet/Prometheus.Client.MetricServer</PackageProjectUrl>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <RepositoryType>git</RepositoryType>
+    <RepositoryUrl>https://github.com/PrometheusClientNet/Prometheus.Client.MetricServer</RepositoryUrl>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>../../Prometheus.Client.MetricServer.snk</AssemblyOriginatorKeyFile>
```

Comment:
We should generate new *.snk

Reply:
We might, but what for? Several libraries can share key.